modules/core/src/lib/attribute.js

```diff
@@ -79,14 +79,16 @@ export default class LayerAttribute extends Attribute {
   setNumInstances(numInstances) {
     const state = this.userData;
 
-    if (!state.isExternalBuffer) {
-      // Do we need to reallocate the attribute's typed array?
-      const instanceCount = this.getInstanceCount();
-      const needsAlloc = instanceCount === 0 || instanceCount < numInstances;
-      if (needsAlloc && (state.update || state.accessor)) {
-        state.needsAlloc = true;
-        this.setNeedsUpdate(this.id);
-      }
+    if (state.isExternalBuffer || state.isCustomBuffer) {
```

Comment:
Is this the only place this new state is used? How about just use the `noAlloc` flag?

Reply:
yeah, `noAlloc` should work fine.